audio_clip/model/esresnet/attention.py

```diff
@@ -0,0 +1,58 @@
+import torch
+import torch.nn.functional as F
+from torch import Tensor
+
+from typing import Tuple
+
+
+class EvilBatchNorm2d(torch.nn.BatchNorm2d):
+    """Disables usage of running statistics when in eval mode.
+    This is necessary to maintain the same behavior in train and eval mode; probably due to a mistake during training.
+    """
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.running_var = None
```

Comment:
I am probably confused but didn't we want to use those value in eval mode?

Reply:
Nop that's the inverse by default in eval mode we are using these values. Here they do not represent enough the dataset distribution bc of some training issue so we don't want to use them.